utils/convert_enumofile_to_graphing.py

```diff
@@ -2,13 +2,16 @@
 import argparse
 import os.path
 from collections import Counter
+import numpy as np
```

Comment:
If numpy is only used to sum a list of integers and then do a log base 2, I would recommend not having it as a dependency. Even though numpy is already a dependency for the main script, sometimes its useful to have these helper scripts be more portable.